db/pg/index.js

```diff
@@ -12,15 +12,23 @@ const { DB } = require('config');
 const MODULE_ID = path.relative(process.cwd(), module.id) || require(path.resolve('./package.json')).name;
 const log = new Logger({source: MODULE_ID});
 
+const sslSetUp = {
+	ssl: true,
+	extra: {
+		ssl: {
+			rejectUnauthorized: false,
+		},
+	}
+}
+
 let db;
 
 module.exports = exports = async (options = DB) => {
 	if (!db || options !== DB) {
 		log.info(`${MODULE_ID} creating new DB instance with options => `, options);
 
 		if (options.uri) {
-			const conn = Object.assign({ ssl: { rejectUnauthorized : false } }, pgConn.parse(options.uri));
-
+			const conn = Object.assign({ ...sslSetUp}, pgConn.parse(options.uri));
```

Comment:
**Question**: Are we expecting the addition to the target object of the following?

from:

```js
{ 
    ssl: { 
        rejectUnauthorized : false 
    }  
}
```

to: 

```js
{
    ssl: true,
	extra: {
		ssl: {
			rejectUnauthorized: false,
		},
	}
} 
```